priv/prelude/file.specs.erl

```diff
@@ -0,0 +1,31 @@
+-module(file).
+
+%% This module contains specs to replace incorrect or inexact specs in OTP.
+
+-type deep_list() :: [char() | atom() | deep_list()].
+-type name_all()  :: string() | atom() | deep_list() | (RawFilename :: binary()).
+-type posix() ::
+        'eacces' | 'eagain' |
```

Comment:
Is it necessary to override posix()? I think it is defined in this module in all supported OTP versions. Then it doesn't need to be defined. It can be used anyway.